app/assets/stylesheets/partials/_modal.scss

```diff
@@ -0,0 +1,84 @@
+#modalOverlay {
+  background-color: white;
+  display: none; 
+  height: 100%;
+  left: 0;
+  margin: 0;
+  opacity: 0.7;
+  padding: 0;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  z-index: 9999;
+}
+
+#modal {
+  background-color: white;
+  border: 2px solid #C9D4D9;
```

Comment:
Color literals like `#C9D4D9` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#C9D4D9` should be written as `#c9d4d9`